src/lake/tests/init/test.sh

```diff
@@ -77,6 +77,15 @@ $LAKE -d lean-data exe lean-data
 $LAKE new 123-hello
 $LAKE -d 123-hello exe 123-hello
 
+# Test creating packages with components that contain `.`s
+# https://github.com/leanprover/lean4/issues/2999
+  
+# this fails on windows for unrelated reasons
+if [ "$OSTYPE" != "msys" ]; then
```

Comment:
@tydeu: I couldn't work this one out: but I figure that whatever this bug is, it's not in scope for this PR.

Reply:
Fair enough. Yeah, I think it can be pushed to a later date.

Reply:
@tydeu: is this PR good to go then?